packages/nitro/src/builders.ts

```diff
@@ -63,10 +63,21 @@ export class LocalBuilder extends BaseBuilder {
       inputFiles,
     });
 
+    const webhookRouteFile = join(this.#outDir, 'webhook.mjs');
+
     await this.createWebhookBundle({
-      outfile: join(this.#outDir, 'webhook.mjs'),
+      outfile: webhookRouteFile,
       bundle: false,
     });
+
+    // Post-process the generated file to wrap with SvelteKit request converter
+    let webhookRouteContent = await readFile(webhookRouteFile, 'utf-8');
```

Comment:
Nitro based builds doesn't need it (nitro already traps errors, i think it should be behind a flag or something for builder)

Reply:
I agree that this should probably go into the builder code itself and `createWebhookBundle` can take an argument called 'addUnhandledRejectionTrap' that defaults to `false` and can be used by builders that need to use it

happy to merge this into my PR still and work on this upstream to simplify

Reply:
and then if only vite needs it as you're saying, then the vite plugin can specifically wire that flag in when using nitro